README.md

```diff
@@ -167,7 +168,36 @@ The fields are as follows:
   sense.
 * `extAttrs`: A list of [extended attributes](#extended-attributes).
 
+### Interface mixins
+
+Interfaces mixins look like this:
+
+```JS
+{
+  "type": "interface-mixin",
```

Comment:
I'm wondering, maybe we should change this to "interface mixin"

Reply:
(that would match "callback interface", and would allow me to use the type directly in ReSpec without needing to special case the type.

Reply:
This is to keep consistency with `line-comment`, `multiline-comment`, etc., although they are undocumented.

Reply:
(Do we want to change them as `line comment`, `multiline comment`, etc.?)

Reply:
Nah, line-comment and multiline-comment are ok.

Reply:
Change made! (But why not `line comment`?)

Reply:
Only because that would be a breaking change (e.g., in ReSpec, I have code that already expects "line-comment").